west.yml: Update hal_stm32 with recent cube packages

FRASTM opened the pull request:
STM32Cube updates:
    stm32cube: update stm32l5 to version V1.4.0
    stm32cube: update stm32l4 to version V1.17.0
    stm32cube: update stm32g4 to version V1.4.0
    stm32cube: update stm32f7 to version V1.16.1
    stm32cube: update stm32f4 to version V1.26.0
    stm32cube: update stm32g0 to version V1.4.1
    stm32cube: update stm32h7 to version V1.9.1
    stm32cube: update stm32mp1 to version V1.4.0

    update stm32cube/common_ll
    
    Signed-off-by: Francois Ramu <francois.ramu@st.com>

erwango commented on west.yml:
```diff
@@ -72,7 +72,7 @@ manifest:
       path: modules/hal/st
     - name: hal_stm32
-      revision: 50283e0438d882a80fb681907664491c7042c817
+      revision: a8498d3aa79595a3ca51ad2dd341d9f28d8343c0
```
```suggestion
      revision: pull/93/head
```
The interest of this method is to be sure we point on and testing the last PR version.

FRASTM replied:
OK, changed to `pull/93/head`

FRASTM commented:
compilation ok with :
 ```
   west build -p auto -b nucleo_g071rb samples/hello_world/
   west build -p auto -b nucleo_f746zg samples/hello_world/
   west build -p auto -b nucleo_h743zi samples/hello_world/
   west build -p auto -b nucleo_f446re samples/hello_world/
   west build -p auto -b nucleo_g474re samples/hello_world/
   west build -p auto -b nucleo_l4r5zi samples/hello_world/
   west build -p auto -b nucleo_l552ze_q samples/hello_world/

```

FRASTM committed:
west.yml: Update hal_stm32 with recent cube packages

STM32Cube updates:

    stm32cube: update stm32l5 to version V1.4.0
    stm32cube: update stm32l4 to version V1.17.0
    stm32cube: update stm32g4 to version V1.4.0
    stm32cube: update stm32f7 to version V1.16.1
    stm32cube: update stm32f4 to version V1.26.0
    stm32cube: update stm32g0 to version V1.4.1
    stm32cube: update stm32h7 to version V1.9.1
    stm32cube: update stm32mp1 to version V1.4.0

    update stm32cube/common_ll

Signed-off-by: Francois Ramu <francois.ramu@st.com>

FRASTM commented:
rebase

erwango commented:
@FRASTM https://github.com/zephyrproject-rtos/hal_stm32/pull/93 has been merged. You can update this PR.

FRASTM commented:
rebase and update as the  zephyrproject-rtos /hal_stm32 is now merged with latest versions of the STM32Cube
